File: Latihan_B/java/Disk.java
public class Disk 
{
    private String type;
    private String capacity;
    private int price;

    // constructor
    public Disk(){}
    public Disk(String type, String capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    // method set dan get untuk type, capacity, dan price
    public void setType(String type) {
        this.type = type;
    }
    public void setCapacity(String capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    public String getType() {
        return type;
    }
    public String getCapacity() {
        return capacity;
    }
    public int getPrice() {
        return price;
    }

    // method print
    public void printDisk() {
        System.out.println("    Disk");
        System.out.println("Type        : " + type);
        System.out.println("Capacity    : " + capacity);
        System.out.println("Price       : " + "Rp. " + price);
        System.out.println("*************************************");
    }
}

File: Latihan_B/php/Disk.php
<?php
class Disk {
    private $type;
    private $capacity;
    private $price;

    // constructor
    function __construct($type, $capacity, $price) {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // method set dan get untuk type, capacity, dan price
    function setType($type) {
        $this->type = $type;
    }
    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }
    function setPrice($price) {
        $this->price = $price;
    }

    function getType() {
        return $this->type;
    }
    function getCapacity() {
        return $this->capacity;
    }
    function getPrice() {
        return $this->price;
    }

    // method print
    function printDisk() {
        echo "  Disk". "<br/>";
        echo "Type        : ".  $this->getType(). "<br/>";
        echo "Capacity    : ". $this->getCapacity(). "<br/>";
        echo "Price       : ". "Rp. ". $this->getPrice(). "<br/>";
        echo "************************************". "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>